official-services.md

```diff
@@ -11,10 +10,7 @@ The following services are officially operated by the FEniCS Project.
 - [PyPi](https://pypi.org/user/fenicsproject/)
 - [Slack](https://fenicsproject.slack.com)
 - [Steering Council mailbox](mailto:fenics-steering-council@googlegroups.com)
```

Comment:
Is this still active?

Reply:
This is stil active and we plan to keep it active